import/index_java/src/org/vufind/index/FormatCalculator.java

```diff
@@ -580,12 +634,20 @@ protected List<String> getFormatsAsList(Record record) {
             result.add("ConferenceProceeding");
         }
 
+        // check the 33x fields; these may give us clear information in newer records
+        // and if we find something here, it currently indicates non-book content.
```

Comment:
Minor thing but this comment could probably be clearer re why couldBeBook is being set to false, so that we reconsider if/when implementing getFormatsFrom33xFields() more fully e.g. 

```
        // check the 33x fields; these may give us clear information in newer records;
        // in current partial implementation of getFormatsFrom33xFields(), if we find 
        // something here, it indicates non-book content.
```

Reply:
Good idea, I've adjusted as you suggested.